PinpointKit/PinpointKit/Sources/Core/FeedbackViewController.swift

```diff
@@ -249,4 +249,36 @@ extension FeedbackViewController: FeedbackTableViewDataSourceDelegate {
         editImageViewController.modalPresentationStyle = feedbackConfiguration?.presentationStyle ?? .fullScreen
         present(editImageViewController, animated: true, completion: nil)
     }
+    
+    @available(iOS 14, *)
+    func feedbackTableViewDataSourceDidRequestScreenshot(feedbackTableViewDataSource: FeedbackTableViewDataSource) {
+        let pickerController = PHPickerViewController(configuration: .init(photoLibrary: .shared()))
+        pickerController.delegate = self
+
+        viewController.present(pickerController, animated: true, completion: nil)
```

Comment:
Any way to limit this to just screenshots via the configuration (none that I’ve seen so far, but maybe there’s an API for a predicate (like we use in screenshot detector) that I’ve missed)? At the very least, we should apply the `PHPickerFilter` for `images`, as you can currently select videos and live photos.

Reply:
I limited it using the filter, but through this API, I'm not sure there's a better way.

Reply:
Yeah, not really.